Snippets/SQSLambda/SQSLambda_0/Usage.cs

```diff
@@ -61,6 +61,26 @@ static void LicenseFile(AwsLambdaSQSEndpointConfiguration endpointConfiguration,
         #endregion
     }
 
+    static void Recoverability(AwsLambdaSQSEndpointConfiguration endpointConfiguration)
+    {
+        #region delayed-retries
+
+        var recoverability = endpointConfiguration.AdvancedConfiguration.Recoverability();
+        recoverability.Delayed(customization =>
+        {
+            customization.NumberOfRetries(5);
+            customization.TimeIncrease(TimeSpan.FromSeconds(15));
+        });
+
+        #endregion
+
+        #region unrestricted-delayed-delivery
+
+        endpointConfiguration.Transport.UnrestrictedDurationDelayedDelivery();
```

Comment:
Looking at the https://docs.particular.net/transports/sqs/delayed-delivery#enable-unrestricted-delayed-delivery
> Unrestricted delayed delivery requires a FIFO queue for each endpoint that receives delayed messages. The transport handles creation of the FIFO queue automatically when installers are enabled.

I think this might not work as we [initialize](https://github.com/Particular/NServiceBus.AwsLambda.Sqs/blob/master/src/NServiceBus.AwsLambda.SQS/TransportWrapper/ManualPipelineInvocationInfrastructure.cs#L10) the AwsEndpoint with NoOpQueueCreator. In other words, I think we should mention that the user needs to create the fifo queue by hand.

Reply:
Added